app/browsered/multi-select.js

```diff
@@ -43,7 +67,7 @@ function progressivelyEnhanceSelects () {
         return { value, id, checked, text }
       })
     }
-    select.outerHTML = multiSelect(configuration)
+    select.outerHTML = nunjucks.renderString(multiSelectTemplate, configuration)
```

Comment:
Could you not just use `nunjucks.render(<template path>)` to render the template file, rather than having to inline it? not particularly important but nicer to not inline it if possible i think

Reply:
I realise this won't work as it's client side, my bad

Reply:
If this is the only usage and we're continuing to look into how we want to handle the build (I believe both are true) then I think this is a reasonable change.

As this seems to be the only case I wonder whether we should consider making an endpoint which renders this template and returns partial HTML to be injected - that way we can more sensibly use things like Design System Components without the complexity spiralling.